app/Main.hs

```diff
@@ -67,6 +75,8 @@ main = do
         decoded <- decoder file
 
         either
-          putStrLn
+          handleError
           (TIO.putStrLn . substitute compiledTemplate . toMustache)
           decoded
+ where
+  handleError err = putStrLn err >> exitFailure
```

Comment:
Do you think `err` should go to `stderr`?

Reply:
Yes. That is now implemented.